apps/transport/priv/gettext/fr/LC_MESSAGES/feedback.po

```diff
@@ -0,0 +1,32 @@
+## "msgid"s in this file come from POT (.pot) files.
+###
+### Do not add, change, or remove "msgid"s manually here as
+### they're tied to the ones in the corresponding POT file
+### (with the same domain).
+###
+### Use "mix gettext.extract --merge" or "mix gettext.merge"
+### to merge POT files into PO files.
+msgid ""
+msgstr ""
+"Language: fr\n"
+"Plural-Forms: nplurals=2; plural=(n>1);\n"
+
+#, elixir-autogen, elixir-format
+msgid "Send the feedback"
+msgstr "Envoyer l’avis"
+
+#, elixir-autogen, elixir-format
+msgid "Why?"
+msgstr "Pourquoi ?"
+
+#, elixir-autogen, elixir-format
+msgid "Your email (optional)"
+msgstr "Votre adresse email (facultatif)"
+
+#, elixir-autogen, elixir-format
+msgid "what’s your feedback on this page?"
```

Comment:
Majuscule initiale pour la phrase en EN.

Peux-tu vérifier si on doit utiliser `’` ou `'` en EN ?

Reply:
> Peux-tu vérifier si on doit utiliser ’ ou ' en EN ?

Et c’est… compliqué. https://graphicdesign.stackexchange.com/questions/66809/what-s-the-right-character-for-an-apostrophe 

> If the text is digital and to be consumed by human eyes, both (the curly and the straight apostrophe) are accepted as an apostrophe and quotation mark. Here schools differ and schism rules. We keep on with the expressive mess. Just be consistent within the same document.

Étant donné que pour le reste du site, les traductions anglaises utilisent l’apostrophe droite, j’ai remis l’apostrophe droite.